packages/eslint-checker/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@stryker-mutator/eslint-checker",
+  "version": "6.0.0-beta.0",
+  "description": "An eslint checker plugin to be used in Stryker, the JavaScript mutation testing framework",
+  "main": "dist/src/index.js",
+  "type": "module",
+  "keywords": [
+    "eslint",
+    "stryker",
+    "checker",
+    "plugin"
+  ],
+  "author": "Tim Lehner <timothy.lehner.2011@live.rhul.ac.uk>",
+  "homepage": "https://github.com/TimLehner/stryker-js#readme",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TimLehner/stryker-js.git"
+  },
+  "scripts": {
+    "test": "c8 npm run test:all",
+    "mocha": "mocha",
+    "test:all": "npm run test:unit && npm run test:integration",
+    "test:unit": "mocha 'dist/test/unit/**/*.spec.js'",
+    "test:integration": "mocha 'dist/test/integration/**/*.js'",
+    "stryker": "node ../core/bin/stryker.js run"
+  },
+  "bugs": {
+    "url": "https://github.com/TimLehner/stryker-js/issues"
+  },
+  "dependencies": {
+    "@stryker-mutator/api": "^6.0.0-beta.0",
+    "@stryker-mutator/util": "6.0.0-beta.0",
+    "cosmiconfig": "^7.0.1",
```

Comment:
@nicojs (how) do we want to set these dependency version? As any part of this major version? Or fixed to a minor/patch version?